.buildkite/hooks/post-command

```diff
@@ -23,3 +24,6 @@ if [ -d "/tmp/artifacts/test-summaries" ] && [ "$(ls -A /tmp/artifacts/test-summ
     echo "--- Cleaning up"
     docker run --rm -v /tmp/artifacts:/artifact-mount alpine:latest /bin/sh -c 'rm -rf /artifact-mount/test-summaries' || true
 fi
+
+# clean up bazel logs
```

Comment:
one last change: could you add a comment here that says something like: "this only has effect when the bazel test runs in the same environment as the buildkite job commands, and has no effect when the tests run inside another test container" ?

many thanks!